taskcluster/firefox_tv_taskgraph/email.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+
+from taskgraph.transforms.base import TransformSequence
+
+
+transforms = TransformSequence()
+
+
+NOTIFY_EMAIL_ADDRESS = 'firefox-tv@mozilla.com'
+
+
+@transforms.add
+def email_task(config, tasks):
+    tag = os.environ['GIT_TAG']
```

Comment:
Bug: This line will break on any decision task that doesn't define GIT_TAG. I'd put:

```suggestion
    tag = os.environ.get('GIT_TAG', "UNDEFINED_TAG")
```

This way, if `target_tasks.py` is bugged and outputs this task in a PR, let's say, the email would be clear something is odd.